assets/javascripts/discourse/components/query-result.js.es6

```diff
@@ -29,8 +30,19 @@ const QueryResultComponent = Ember.Component.extend({
   columns: Em.computed.alias('content.columns'),
   params: Em.computed.alias('content.params'),
   explainText: Em.computed.alias('content.explain'),
-
   hasExplain: Em.computed.notEmpty('content.explain'),
+
+  @computed('content.result_count')
+  resultCount: function(count) {
+    if (count === 1) {
+      return I18n.t('explorer.single_result_count', {value: count});
+    } else if (count === 250) {
```

Comment:
Where is this magic number coming from?

Reply:
It comes from here :face_with_head_bandage:

https://github.com/discourse/discourse-data-explorer/blob/68059b540ee3af199e4f6aca0beca19e8597a350/plugin.rb#L102